packages/app-desktop/gui/ToolbarButton/ToolbarButton.tsx

```diff
@@ -43,7 +43,10 @@ export default function ToolbarButton(props: Props) {
 	if (!isEnabled) classes.push('disabled');
 
 	const onClick = getProp(props, 'onClick');
-
+	const style: any = {
```

Comment:
Please don't use "any". `React.CSSProperties` is the correct type here I believe